style.css

```diff
@@ -4,35 +4,48 @@
 
 body {
     box-sizing: border-box;
-    padding-top: 2rem;
+    padding-top: 6rem;
     display: none;
```

Comment:
OPTIONAL
You might want to crosscheck this property, I had to remove this line to review your latest changes.

Reply:
Do you mean `padding-top` ?
What is wrong with it?

Reply:
`display: none;`

Currently, your page is blank due to this property.

Reply:
I have hidden the desktop view. 

https://github.com/backtofayton/portfolio/pull/1#issue-1094520412

Reply:
Should I merge it now?